### ISSUE TITLE
Login Blocked in Blaze.com

### ISSUE THREAD

**Issue by ThomasTrovon** [ultrafunkamsterdam/undetected-chromedriver#914]
Hi Friends.. 

Can anyonelse login into blaze using undetected-chromedriver? Because with a regular browser I can. But when I use undetected-chromedriver I stay blocked in login screen, even when I complete the CAPTCHA. I tried with v1 and v2 with the below codes:

V1
```
import undetected_chromedriver as uc
from subprocess import CREATE_NO_WINDOW


# Função para abrir o navegador, conectar na blaze, e retornar o driver com a conexão para operações de leitura
def conectar():
    url = 'https://blaze.com/pt/games/ruleta-en-vivo'

    driver = uc.Chrome(service_creationflags=CREATE_NO_WINDOW, use_subprocess=True)
    driver.get(url)

    return driver
```

V2
```
import undetected_chromedriver.v2 as uc
from subprocess import CREATE_NO_WINDOW


# Função para abrir o navegador, conectar na blaze, e retornar o driver com a conexão para operações de leitura
def conectar():
    url = 'https://blaze.com/pt/games/ruleta-en-vivo'

    driver = uc.Chrome(service_creationflags=CREATE_NO_WINDOW, use_subprocess=True)
    driver.get(url)

    return driver
```

If I doing something wrong, can anyone tell me what is, please?
Or, "the fault" is of this site has a detection script very strong?

**Comment by sebdelsol** [comment]
`service_creationflags=subprocess.CREATE_NO_WINDOW` won't help you with detection. 
It's just a way to prevent Selenium from opening a command line window when the driver is launched.

You are detected and that seems bad, but I can't I can't reproduce your issue.

**Comment by ThomasTrovon** [comment]
> `service_creationflags=subprocess.CREATE_NO_WINDOW` won't help you with detection. It's just a way to prevent `Selenium` from opening a command line window when the driver is launched.
> 
> You are detected and that seems bad, but I can't reproduce your issue.

Yes friend. I know that `service_creationflags=subprocess.CREATE_NO_WINDOW` won't help me with detection. I just paste here how the code was in my file. The real reason I use this is really prevent Selenium from opening a command line window when the driver is launched in my bot. This part can simply be ignored. Ready. 

I removed it above, because it doesn't make a difference now.

So with the simple code:

```
import undetected_chromedriver as uc

def conectar():
    url = 'https://blaze.com/pt/games/ruleta-en-vivo'

    driver = uc.Chrome()
    driver.get(url)

    return driver
```

I have the error. Both: in v1 and v2.

You said that you can't reproduce my issue. So, to this, I create a fake account, that, if you have time to reproduce this, please, you can use to see the error:

Login: thomastrovon@gmail.com
Password: Fakeaccount10

If you could help me, if have time for it, just login in a regular browser. Then after, try to login in a UC browser. You will see the difference.

If you, or anyone help me to solve my problem, as a thank you, I will share with who help me to solve this, with freely forever access to my bot telegram channels of my signals, if you want to receive it as a thank you gift... <3

**Comment by ThomasTrovon** [comment]
@sebdelsol 

Friend.. Here I make a video demonstrating the problem:

https://www.youtube.com/watch?v=MWCkuN0eZHQ

**Comment by Martinzz** [comment]
Me too, some other sites

**Comment by ThomasTrovon** [comment]
> Me too, some other sites

So, no solution yet? :C

So sad... :'(

**Comment by oguidomingos** [comment]
Are you using a mac?  I was facing similar problems using puppeteer for executing a macro script, changed the code into a Windows and I could login easily.

**Comment by ThomasTrovon** [comment]
> Are you using a mac? I was facing similar problems using puppeteer for executing a macro script, changed the code into a Windows and I could login easily.

No Bro. How can you see in the video

https://www.youtube.com/watch?v=MWCkuN0eZHQ

I'm already using Windows.. :(
